README.md

```diff
@@ -166,6 +173,37 @@ Once you initialized the two keyspaces, fill in the configuration values in `val
 ~/temporal-helm$ helm install -f values/values.cassandra.yaml temporaltest . --timeout 900s
 ```
 
+### Install and configure Temporal
+
+In a live environment, where all dependencies are already hardened and operational, it's possible to install only the Temporal server components with this Helm chart. Whether using MySQL or Cassandra, the preceeding steps to configure schema should be followed. This example assumes Cassandra, demonstrates setting values on the commandline rather than via environment, and enables TLS for the database connection:
```

Comment:
This line is a bit confusing:
> "In a live environment, where all dependencies are already hardened and operational, it's possible to install only the Temporal server components with this Helm chart"

It sounds like we are trying to get the point across that, IF an application environment already uses certain systems that Temporal uses as dependencies, then those systems can continue to be used, and this Helm chart can install the minimal pieces of Temporal such that Temporal can then be configured to use those systems as its dependencies ?

Reply:
yes... do you have a better/shorter way to say all that?

Reply:
What about:

```suggestion
If a live application environment already uses systems that Temporal can use as dependencies, then those systems can continue to be used. This Helm chart can install the minimal pieces of Temporal such that it can then be configured to use those systems as its dependencies.

The example below demonstrates a few things:
1. How to set values via the command line rather than the environment.
2. How to configure a database (shows Cassandra, but MySQL works the same way)
3. How to enable TLS for the database connection.
```